src/test/java/com/expedia/pitchfork/systems/common/SpanValidatorTest.java

```diff
@@ -1,4 +1,4 @@
-package com.expedia.pitchfork.systems.haystack;
+package com.expedia.pitchfork.systems.common;
```

Comment:
Minor: there is a lot of package moving in this pull request, from `haystack` to `common`. Maybe worth mentioning it in the PR's description and the reason why you are doing it?

Reply:
yeap, done